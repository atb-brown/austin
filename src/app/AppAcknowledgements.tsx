import React from "react";
import Link from "../component/Link";

/**
 * Acknowledgements for individuals and projects that made this app possible.
 *
 * @return {ReactElement}
 */
export default function AppAcknowledgements() {
  const period = ".";

  return (
    <div>
      <div className="App-soft-text" style={{ fontSize: 20, paddingLeft: 50 }}>
        <Link
          description="Initialized"
          url="https://github.com/atb-brown/austin/commit/5c8514a96194e6271d4e8e7f234499ee4e95ab3d"
        />
        {" using "}
        <Link
          description="create-react-app"
          url="https://create-react-app.dev/"
        />{" "}
      </div>
      <div className="App-soft-text" style={{ fontSize: 20 }}>
        {"This website is deployed by borrowing the incredible efforts of the "}
        <Link
          description="react-gh-pages contributors"
          url="https://github.com/gitname/react-gh-pages/graphs/contributors"
        />
        {period}
      </div>
    </div>
  );
}
