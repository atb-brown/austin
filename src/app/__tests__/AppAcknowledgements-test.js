import renderer from "react-test-renderer";
import AppAcknowledgements from "../AppAcknowledgements";

it("renders to match snapshot", () => {
  const tree = renderer.create(<AppAcknowledgements />);

  expect(tree).toMatchSnapshot();
});
