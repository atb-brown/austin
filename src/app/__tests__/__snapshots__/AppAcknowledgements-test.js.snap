// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders to match snapshot 1`] = `
<div
  style={
    {
      "alignItems": "flex-end",
      "display": "flex",
      "flexDirection": "column",
      "justifyContent": "flex-end",
      "marginRight": "3%",
      "width": "100%",
    }
  }
>
  <div
    className="App-soft-text"
    style={
      {
        "fontSize": 20,
        "paddingLeft": 50,
      }
    }
  >
    <a
      className="App-link"
      href="https://github.com/atb-brown/austin/commit/5c8514a96194e6271d4e8e7f234499ee4e95ab3d"
    >
      Initialized
    </a>
     using 
    <a
      className="App-link"
      href="https://create-react-app.dev/"
    >
      create-react-app
    </a>
     
  </div>
  <div
    className="App-soft-text"
    style={
      {
        "fontSize": 20,
      }
    }
  >
    This website is deployed by borrowing the incredible efforts of the 
    <a
      className="App-link"
      href="https://github.com/gitname/react-gh-pages/graphs/contributors"
    >
      react-gh-pages contributors
    </a>
    .
  </div>
</div>
`;
